opr/functions.py

```diff
@@ -49,3 +49,23 @@ def gc_clamp_calc(sequence):
     if len(sequence) < 5:
         return 0
     return sequence[-5:].count('G') + sequence[-5:].count('C')
+
+def run_length(sequence, nucleic_acid):
```

Comment:
I suggest renaming this function to `single_run_length` because, in the next version, we have a `double_run_length`.